ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -446,36 +449,124 @@ def test_full_backup_reinstall_restore_with_vault(self):
 
 
 class TestBackupAndRestoreWithReplica(IntegrationTest):
-    """Regression test for https://pagure.io/freeipa/issue/7234"""
-    num_replicas = 1
+    """
+        Regression tests for issues 7234 and 7455
+
+        https://pagure.io/freeipa/issue/7234
+            - check that oddjobd service is started after restore
+            - check new replica  setup after restore
+        https://pagure.io/freeipa/issue/7455
+            check that after restore and replication reinitialization
+                - users and CA data are at state before backup
+                - CA can be installed on existing replica
+                - new replica with CA can be setup
+    """
+    num_replicas = 2
     topology = "star"
+    csr = "my.csr"
 
     @classmethod
     def install(cls, mh):
+        cls.replica1 = cls.replicas[0]
+        cls.replica2 = cls.replicas[1]
         if cls.domain_level is None:
             domain_level = cls.master.config.domain_level
         else:
             domain_level = cls.domain_level
 
-        if cls.topology is None:
-            return
-        else:
-            tasks.install_topo(
-                cls.topology, cls.master, [],
-                cls.clients, domain_level
-            )
+        # Configure only master and one replica.
+        # Replica is configured without CA
+        tasks.install_topo(
+            cls.topology, cls.master, [cls.replica1],
+            cls.clients, domain_level,
+            setup_replica_cas=False
+        )
+
+    def prepare_cert_signing_request(self):
+        self.master.run_command(
+            ['ipa', 'service-add', 'TEST/' + self.master.hostname])
+        openssl_command = ['openssl', 'req', '-new', '-nodes',
+                           '-out', self.csr,
+                           '-subj', '/CN=' + self.master.hostname]
+        self.master.run_command(openssl_command)
+        self.replica1.run_command(openssl_command)
+        self.replica2.run_command(openssl_command)
+
+    @staticmethod
```

Comment:
We generally don't use ``staticmethod``. I know that some editors tell you that you can add ``staticmethod`` decorator. Please ignore your IDE. It's a stupid suggestion. There is no benefit and just draw backs.